plugins/modules/snap.py

```diff
@@ -47,10 +50,12 @@
         type: str
     classic:
         description:
-            - Confinement policy. The classic confinement allows a snap to have
-              the same level of access to the system as "classic" packages,
-              like those managed by APT. This option corresponds to the C(--classic) argument.
-              This option can only be specified if there is a single snap in the task.
+            - Confinement policy: This level of confinement is permissive, granting full system access,
+              similar to that of traditionally packaged applications, such as Debian packages (those managed via the C(apt) command-line tool).
+              This option corresponds to the C(--classic) argument of the C(snap install) command.
+              It can only be specified when the task involves a single snap.
```

Comment:
I would prefer us not to mention a specific OS as example. `snap` can be used in RHEL, OpenSUSE and other distros as well. I think it is better to keep the wording neutral.

Reply:
> This explanation essentially mirrors the wording used in the snapcraft.io documentation: [snapcraft.io/docs/classic-confinement](https://snapcraft.io/docs/classic-confinement). I just wanted to clarify what it means to install using classic confinement, but I will expand on it by mentioning that this option installs a snap with classic confinement **AND** leave the sentence explaining what this entails.

hi @locnnil , while I appreciate your intention, I think we must be aware that the purpose of this documentation is NOT to explain how `snap`-command works, but rather how to use the `snap`-ansible-module. Someone who does not know how to use snap is unlikely going to use this documentation to learn it. So, it is more than OK to add references and links to more documentation about the `snap` command, but too much info here might not make things better.

(Just occurred me a couple of minutes later: if you are mirroring the wording in existing documentation, it might be more interesting to point to that one rather than copy-paste it. When copying the text, there is an implicit task added to our list of checking the source doc for updates and keeping the module docs synchronized. It's so much easier to just tell the user: read it over there!)

> @russoz, I will revise the wording to "traditionally packaged applications that don't use sandboxing mechanisms" and remove the part about Debian packages. Would that work for you?

I use a couple snaps locally, I am far from being an expert on it. Although I can infer the meaning of this, based on experience, and of course we can google it (but I don't want to spend my time on that), I reckon

Reply:
@russoz I removed some of the parts that explains more technically, can you give me your feedback on this now please?